OverlayPlugin.Core/OverlayZCorrector.cs

```diff
@@ -23,19 +24,25 @@ public OverlayZCorrector(TinyIoCContainer container)
 
             var span = TimeSpan.FromSeconds(3);
             timer = new Timer(EnsureOverlaysAreOverGame, null, span, span);
+
+            repository.RegisterProcessChangedHandler(UpdateFFXIVProcess);
```

Comment:
My memory is that this doesn't get called if the process has already been set.  See: quisquous/cactbot#4093.

Reply:
To clarify, this is actually more of a bugfix. The removed code was dummied out (`return`'d immediately) anyways.

I guess a more minimal "fix" for the build warnings would have just been to remove the entire block of code after the return, but then there's this potential bug sitting out there with no indication.

I opted for hooking up the event listener the same way that it was used elsewhere in OP, and if the event not being called is an issue, it can be addressed across the project.

Reply:
Sounds good, I'll file an issue.